vignettes/adrs.Rmd

```diff
@@ -545,9 +582,7 @@ adrs <- adrs %>%
     dataset_ref = adsl,
     dataset_add = adrs,
     by_vars = exprs(STUDYID, USUBJID),
-    filter_add = PARAMCD == "BOR" & AVALC %in% c("CR", "PR") & ANL01FL == "Y",
-    order = exprs(ADT, RSSEQ),
-    mode = "first",
+    filter_add = PARAMCD == "BOR" & AVALC %in% c("CR", "PR"),
```

Comment:
same update needed for adrs_basic?

Reply:
It does not affect the results. It just makes the code a bit shorter. I have updated adrs_basic and the templates.